data/human/human missions.txt

```diff
@@ -3996,3 +3996,281 @@ mission "Fumbling Mugger"
+	on complete
+		payment 20000 500
+		conversation
+			`As before on Skymoot, Karengo guides you down to a landing spot far away from the local government. "Don't want no interruptions," he says.`
+			`	Despite days of pressuring, you've been unable to get Karengo to tell you exactly what a quoijuu is, other than the slightly ominous, "the biggest fish". He sends a file to your ship computer and grins. "Since it's too late to back out now, it's time to fill you in. Quoijuu aren't the only thing down there; you've also got scyllae, taniwhales, morgawrii, plus some hydrothermal vents, none of which will benefit us if we run into them."`
+			`	"The good news is that there is a path to follow. The bad news is that the only person to make the path public was a bit of a nut. It's all in riddles, and I haven't been able to make head nor tail of them. Look them over and we'll follow your lead. And don't worry too much about it - what's the worst that could happen?"`
+			choice
+				`	"Ok, I'll take a look."`
+					goto adventure
+				`	"We could get lost and then die."`
+			`	Karengo smiles in a way that is only reminiscent of reassuring.`
+			label adventure
+			`	The adventure seekers, who you've spent the trip getting to know, are getting ready. You watch a nearby pair, Daciana and Mussie, helping each other out to put on wetsuits, when Daciana catches your gaze and grins. "Bafta, mate. Safe sailing, see you on the other side." Karengo gives you a signal and you hurry back to the cockpit to fly out, away from safe shoals and over the watery abyss.`
+			`	At your touch, the ship slowly sinks below the waves. As Karengo warned you about on the way over, most of your navigation systems quickly flash red warnings and go offline; there is some unknown interference that overwhelm your ship's sensors, which were designed for space first and water second, or fifth, or not at all. Minutes tick by as the crew silently watches one of the few sensors still working. A young couple, Balazs and Jorma, uneasily exchange glances.`
+			`	EIGHT KILOMETERS, the depth meter reads.`
+			`	TEN KILOMETERS.`
+			`	SIXTEEN KILOMETERS. A loud, metallic creak echoes through the ship and Mussie jumps, startled.`
+			`	Karengo claps him on the back and gives him a signature grin. "Maybe <first> can keep you aboard after this in case they needs anything from a high shelf!" Mussie smiles weakly, but his eyes don't leave the readout.`
+			`	TWENTY-FIVE KILOMETERS. "This's it," Karengo announces, "Sub up! Let's see what there is to see. And <first>, keep it to passive sensors. Don't want to attract any undue attention." The ship smoothly descends into the mouth of a massive cave and your visibility, clouded by dark sulfide emissions from the hydrothermal vents at the floor, drops to less than eight meters. A fan of 10 personal subpods fan out behind you, their communication equipment syncing with yours to display them as blips on your screen.`
```

Comment:
```suggestion
			`	TWENTY-FIVE KILOMETERS. "This's it," Karengo announces, "Sub up! Let's see what there is to see. And <first>, keep it to passive sensors. Don't want to attract any undue attention." The ship smoothly descends into the mouth of a massive cave and your visibility, clouded by dark sulfide emissions from the hydrothermal vents at the floor, drops to less than eight meters. 10 personal subpods fan out behind you, their communication equipment syncing with yours to display them as blips on your screen.`
```

Reply:
Seems kind of repetitive of the word fan.

Reply:
Nevermind I can't read.